README.md

```diff
@@ -12,57 +16,56 @@ Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+Mock server is acessible on [http://localhost:3456](http://localhost:3456).
 
-Instead, it will copy all the configuration files and the transitive dependencies (Webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+The page will reload if you make edits.
+You will also see any lint errors in the console.
 
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+### `npm test`
 
-## Learn More
+Launches the test runner in the interactive watch mode.
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+### `npm run test:ci`
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+Launches the test runner and collects coverage information separately for each test.
 
-### Code Splitting
+### `npm run test:update`
 
-This section has moved here: https://facebook.github.io/create-react-app/docs/code-splitting
+Updates all snapshots for tests.
 
-### Analyzing the Bundle Size
+### `npm run build`
 
-This section has moved here: https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size
+Builds the app for production to the `build` folder.
+It correctly bundles React in production mode and optimizes the build for the best performance.
 
-### Making a Progressive Web App
+The build is minified and the filenames include the hashes.
+Your app is ready to be deployed!
 
-This section has moved here: https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app
+See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
 
-### Advanced Configuration
+## Configuring API URL
+You can configure your API URL during start or build process.
 
-This section has moved here: https://facebook.github.io/create-react-app/docs/advanced-configuration
+```bash
+REACT_APP_API_URL=$YOUR_API_URL npm start
+or
+REACT_APP_API_URL=$YOUR_API_URL npm run build
+```
 
-### Deployment
+API URL is set to `http://localhost:3456` by default. You can change that address by providing proper value of `REACT_APP_API_URL` env variable
 
-This section has moved here: https://facebook.github.io/create-react-app/docs/deployment
+## Deploy on AWS
 
-### `npm run build` fails to minify
+You can simply use aws-cli to deploy App. Just run above command in the project directory (after succesfull build):
```

Comment:
if you are using `aws-cli` it would be handy to add link to it - just to make it clear - at least someone without appropriate knowledge pointed to external `aws` documentation would be able to deploy it without searching too much.

Reply:
adding prerequisites would be helpful as well.